operator/v1/types_network.go

```diff
@@ -318,6 +318,14 @@ type KuryrConfig struct {
 	// +kubebuilder:validation:Minimum=0
 	// +optional
 	PoolBatchPorts *uint `json:"poolBatchPorts,omitempty"`
+
+	// mtu is the MTU that Kuryr should use when creating pod networks in Neutron.
+	// The value has to be lower or equal to the MTU of the nodes network and Neutron has
+	// to allow creation of tenant networks with such MTU. If unset Pod networks will be
+	// created with the same MTU as the nodes network has.
+	// +kubebuilder:validation:Minimum=0
```

Comment:
what happens if you actually specify something like one.  That doesn't seem likely to do good things.

Reply:
Even though there is little sense, Neutron allows to create Networks/Ports with MTU 1.
But we can always try to enforce the minimum value allowed on the cluster-network-operator as other SDNs do [1].

[1] https://github.com/openshift/cluster-network-operator/blob/master/pkg/network/openshift_sdn.go#L110-L112